integration-cli/docker_cli_events_test.go

```diff
@@ -172,26 +181,6 @@ func (s *DockerSuite) TestEventsContainerEventsAttrSort(c *check.C) {
 	c.Assert(matchedEvents, checker.Equals, 2)
 }
 
-func (s *DockerSuite) TestEventsContainerEventsSinceUnixEpoch(c *check.C) {
```

Comment:
`TestEventsContainerEventsSinceUnixEpoch` runs with `--since` and specifies the UNIX epoch as the start time. I don't see `TestEventsTimestampFormats` doing that?